spec/Section 2 -- Language.md

```diff
@@ -214,6 +215,19 @@ mutation {
 }
 ```
 
+This subscription operation will retrieve the current number of likes and create
```

Comment:
This bit is over-specified. A subscription doesn't have to do a retrieve then subscription, it probably only creates the subscription.